### ISSUE TITLE
Getting 200 response but not getting any response

### ISSUE THREAD

**Issue by anijadhav** [cowinapi/developer.cowin#264]
@cowinapi Trying on postman with user agent mentioned in SS but still not getting any response from API. PFB  SS
![image](https://user-images.githubusercontent.com/51171320/118618324-20c15d80-b7e1-11eb-83e2-8bed2d53dad4.png)

**Comment by siddhesh1770** [comment]
Your request is working fine, Blank List means there are no slots available. And also use tomorrow's date because usually same date slots are Booked

**Comment by anijadhav** [comment]
@siddhesh1770 So it should give a proper response as "NO SLOTS AVAILABLE" as a warning or error so that it is easy to identify the issue. Just a suggestion....

**Comment by siddhesh1770** [comment]
Yea I Understand but this is how server response further now you can design your code to show messages like this.
You can check out my Repo for CoWin Tracker on [https://github.com/siddhesh1770]()

And please follow if you like this one I'm going to publish a bot that can book appointment fully automatic except captcha

**Comment by kichappa** [comment]
> @siddhesh1770 So it should give a proper response as "NO SLOTS AVAILABLE" as a warning or error so that it is easy to identify the issue. Just a suggestion....

No, no. The empty array is put to make it consistent with any code that utilises it. If it raises an error, then all loops over this array will break. You may use `calendarByPin` to get details of empty slots, if you prefer.
